### PR TITLE
Add Pylons chain

### PR DESCRIPTION
**Pull request by romanv1812**

### CODE REVIEW

File: pylons/assetlist.json [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+          "denom": "rock",
+          "exponent": 6
+        }
+      ],
+      "base": "ubedrock",
+      "name": "Pylons",
+      "display": "rock",
+      "symbol": "ROCK",
+      "logo_URIs": {
+        "png": "https://raw.githubusercontent.com/Crouton-Digital/Chain-registry/master/pylons/images/pylons.png"
```
Comment (JeremyParish69) [inline pylons/assetlist.json:22]: Please point to the cosmos' Chain Registry here, not a fork

File: pylons/chain.json [hunk 2]
```diff
@@ -0,0 +1,95 @@
+{
+  "$schema": "../chain.schema.json",
+  "chain_name": "pylons",
+  "status": "live",
+  "network_type": "mainnet",
+  "website": "https://pylons.tech",
+  "pretty_name": "Pylons",
+  "chain_id": "pylons-mainnet-1",
+  "bech32_prefix": "pylo1",
```
Comment (JeremyParish69) [inline pylons/chain.json:9]: Please register a prefix https://github.com/satoshilabs/slips/blob/master/slip-0173.md

File: pylons/assetlist.json (hunk 1 above)
Comment (JeremyParish69) [reply pylons/assetlist.json:22]: (and for all cases where you have an image)

File: pylons/chain.json (hunk 2 above)
Comment (romanv1812) [reply pylons/chain.json:9]: I have submitted a pull request to add the prefix.

File: pylons/assetlist.json (hunk 1 above)
Comment (romanv1812) [reply pylons/assetlist.json]: fixed

File: pylons/chain.json (hunk 2 above)
Comment (romanv1812) [reply pylons/chain.json:9]: > Please register a prefix https://github.com/satoshilabs/slips/blob/master/slip-0173.md

https://github.com/satoshilabs/slips/pull/1665

Comment (JeremyParish69) [reply pylons/chain.json:9]: Prefix should be changed here to just 'pylo' then.

Comment (romanv1812) [reply pylons/chain.json]: > Prefix should be changed here to just 'pylo' then.

done

**Pull request merged**